### ISSUE TITLE
Flycheck, while using pylint, is reporting double errors

### ISSUE THREAD

**Issue by mahmoudimus** [flycheck/flycheck#495]
Here's what I see:

![image](https://cloud.githubusercontent.com/assets/40820/4471565/def72990-493b-11e4-8331-a5089cf02a8c.png)

A test python file:

``` python
#!/usr/bin/env python
import os
import abc

print os.path

import flask

for var in collection:
    play


def foo():
    pass


class Object(object):

    def __init__(self, a, b, c, de, f, g, h, j, k):
        pass

```

Here's what happens when I run this from the command line:

![image](https://cloud.githubusercontent.com/assets/40820/4471570/fe57158e-493b-11e4-8cd6-db5cdc0467f8.png)

Here's what I'm configuring flycheck with: https://github.com/mahmoudimus/.emacs.d/blob/exp/personal/core-python.el.inc#L234-L258

Am I missing something stupid here?
